alerts/openshift-virtualization-operator/VirtualMachineStuckInUnhealthyState.md

```diff
@@ -0,0 +1,303 @@
+# VirtualMachineStuckInUnhealthyState
+
+## Meaning
+
+This alert fires when a VirtualMachine has been stuck in an unhealthy state
+for more than 10 minutes and does not have an associated VMI
+(VirtualMachineInstance).
+
+The alert indicates that a VirtualMachine is experiencing early-stage
+lifecycle issues before a VMI can be successfully created.
+This typically occurs during the initial phases of VM startup when OpenShift
+Virtualization
+is trying to provision resources, pull images, or schedule the workload.
+
+**Affected States:**
+- `Provisioning` - Resources (DataVolumes, PVCs) are being prepared
+- `Starting` - VM is attempting to start but no VMI exists yet
+- `Terminating` - VM is being deleted but without an active VMI
+- `Error` states - Various scheduling, image, or resource allocation errors
+
+## Impact
+
+- **Severity:** Warning
+- **User Impact:** VMs cannot start or are stuck in error states
+- **Business Impact:** Workloads cannot be deployed, affecting application
+  availability
+
+## Possible Causes
```

Comment:
Not sure if all the items in the lists need to be highlighted (because if everything is highlighted, nothing really is), but that's a minor niggle.